README.md

```diff
@@ -42,7 +8,13 @@ After you're finished please remove all the comments and instructions!
   - [🛠 Built With](#built-with)
     - [Tech Stack](#tech-stack)
     - [Key Features](#key-features)
+    - [Website Mockup 📱 💻 🖥️](#screenshots)
```

Comment:
_note: Since there is no open PR in the frontend repo for me to review, I'll include my comments for that project here:_

# Frontend review

- [ ] Kindly, remove the pieces of commented code present in these files in order to improve the repo's cleanliness:

 - BookingForm.jsx
 - Homepage.jsx
 - MyBookings.jsx
 - reservationSlice.js
 - teachersSlice.js (this one is full of commented code, kindly remove if not using the file).

- [ ]  Please, kindly refactor the functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry: 

![image](https://user-images.githubusercontent.com/57421823/231915073-f64a8784-8e65-4943-9333-38f2cfcbd2de.png)

That way, your code will meet the latest standards 💪 

- [ ] When I try to click on "Booking form", this happens:

![image](https://user-images.githubusercontent.com/57421823/231916559-d7ed0387-d636-495e-8de3-6e62560cd61c.png)

![image](https://user-images.githubusercontent.com/57421823/231916578-59a10b05-273c-48fd-a0f5-cad8548e4180.png)

I get a blank page with console errors.  It seems that "name" is undefined.

If this is a bug, kindly fix is so your app runs as intended. 👍 

- [ ] When I try to click on "Teachers", this happens:

![image](https://user-images.githubusercontent.com/57421823/231917087-99e2f6f2-09e5-412c-8db6-c156b1d5fe10.png)

There are no console errors and the server is never sent a request.  Maybe this hasn't been implemented yet?  I can see 30 teachers in the backend database waiting to be consumed.  

Kindly, either implement this feature or remove the section, in order to prevent confusion.

- [ ] You are a team of 3-4 people, but I couldn't find a way to add or delete courses.  These are core features so kindly implement them to have a better chance on your final evaluation.

_Important tip:  If something happened that prevented you to implement those features, include a clear explanation in your PR description about what happened, and your reasons.  Being able to explain why you missed features before the deadline IS part of the game.  Both here and in the software industry._

- [ ] When I click "Cancel" to remove a reservation, it won't disappear unless I reload the page.  Even if I click multiple times:

### When I click on "cancel":

![image](https://user-images.githubusercontent.com/57421823/231918428-944608fa-e432-42be-9efe-2ce3ea18e2a6.png)

You can see that after the first click, It won't disappear but the server responds with "not found".  This is because on the first click, it works for the server, but the frontend won't update until I reload the page.

### When I finally reload the section the register finally dissapears:

![image](https://user-images.githubusercontent.com/57421823/231918547-56ffa095-69c7-4c64-8554-eb4b2688f2ea.png)

Kindly correct this behavior in order to improve your UX and prevent confusion for your users. 💪 

- [ ] In the mobile view, The "Bookings" table container is overflowing:

![image](https://user-images.githubusercontent.com/57421823/231919516-da76221f-062d-4a1f-8d82-cbc400ac25e4.png)

Kindly, either use flexbox, CSS grid, simple margins/paddings, or some limiter so this container does not overflow the mobile view on this section, because the users need to scroll to see the data.  That way, your bookings table on the mobile view will be informative, and beautiful! 💪